examples/controls/pickPosition_control.js

```diff
@@ -0,0 +1,21 @@
+var pickControl = document.createElement('div');
```

Comment:
lowerCamelCase if I'm not mistaken too. The best is that i choose lowerCamelCase.